### PR TITLE
Added test with large RTT

### PR DESCRIPTION
**Pull request by twdkeule**
I managed adding a test with a large RTT (75ms) and while downloading a large file. But iI am not yet sure whether it triggers issue #624, as it is one large file and not different (<1MB files at a 2s interval (as in the issue).

However, the filesizes of the logs are quite large and the tests quite slow (for localhost)

### CODE REVIEW

File: integrationtests/rtt_test.go [hunk 1]
```diff
@@ -59,8 +64,32 @@ var _ = Describe("non-zero RTT", func() {
...
+	version := protocol.Version35
+	Context(fmt.Sprintf("with quic version %d", version), func() {
+		fileSizes := [...]int{dataLen, 5 * 1024 * 1024, dataLongLen}
+		for _, fileSize := range fileSizes {
+			fileSizeMB := fmt.Sprintf("%.2f", float64(fileSize)/1024/1024)
+			It("gets a "+fileSizeMB+"MB file with 75ms RTT", func() {
+				rtt := 75 * time.Millisecond
+				testname := "./large_rtt_" + rtt.String() + "_Q" + fmt.Sprint(version) + "_" + fileSizeMB + "MB.log"
+
+				f, err := os.Create(testname)
```
Comment (marten-seemann) [inline integrationtests/rtt_test.go]: Remove the logging here.

FYI, when running integrationtests, you can run them with `QUIC_GO_LOG_LEVEL=DEBUG gingko -- -logfile=log.txt`, so you don't have to modify the test code when you want a log.

File: integrationtests/rtt_test.go [hunk 2]
```diff
@@ -59,8 +64,32 @@ var _ = Describe("non-zero RTT", func() {
 
 		Context(fmt.Sprintf("with quic version %d", version), func() {
 			It("gets a file with 10ms RTT", func() {
-				runRTTTest(10*time.Millisecond, version)
+				runRTTTest(10*time.Millisecond, version, 20)
 			})
 		})
 	}
+	version := protocol.Version35
```
Comment (marten-seemann) [inline integrationtests/rtt_test.go]: You probably forgot to remove this line.

**Comment by twdkeule** [comment]
Ah yes, I forgot about the environment variable. I was testing too specific for the issue

File: integrationtests/rtt_test.go (hunk 1 above)
Comment (lucas-clemente) [inline integrationtests/rtt_test.go]: Nit: you don't actually need the env when you set the flag.

Comment (marten-seemann) [inline integrationtests/rtt_test.go]: Oh right. I guess I tried to be too clever when implementing that flag ;)

File: integrationtests/rtt_test.go [hunk 3]
```diff
@@ -59,8 +64,22 @@ var _ = Describe("non-zero RTT", func() {
...
-				runRTTTest(10*time.Millisecond, version)
+				runRTTTest(10*time.Millisecond, version, 20)
 			})
+
+			fileSizes := [...]int{dataLen, 5 * 1024 * 1024, dataLongLen}
+			for _, fileSize := range fileSizes {
+				fileSizeMB := fmt.Sprintf("%.2f", float64(fileSize)/1024/1024)
+				It("gets a "+fileSizeMB+"MB file with 75ms RTT", func() {
+					flags := log.Flags()
+					log.SetFlags(log.Ldate | log.Ltime | log.Lmicroseconds)
```
Comment (lucas-clemente) [inline integrationtests/rtt_test.go]: Remove the 5 logging relate lines here and below – we don't always need the logs and they may skew the tests by slowing them down. We also already globally set the timestamp format for logs.

File: integrationtests/rtt_test.go [hunk 4]
```diff
@@ -1,14 +1,18 @@
 package integrationtests
 
 import (
+	"bufio"
```
Comment (lucas-clemente) [inline integrationtests/rtt_test.go]: Run goimports to remove the unused ones.

File: integrationtests/rtt_test.go [hunk 5]
```diff
@@ -59,8 +64,22 @@ var _ = Describe("non-zero RTT", func() {
 
 		Context(fmt.Sprintf("with quic version %d", version), func() {
 			It("gets a file with 10ms RTT", func() {
-				runRTTTest(10*time.Millisecond, version)
+				runRTTTest(10*time.Millisecond, version, 20)
 			})
+
+			fileSizes := [...]int{dataLen, 5 * 1024 * 1024, dataLongLen}
+			for _, fileSize := range fileSizes {
+				fileSizeMB := fmt.Sprintf("%.2f", float64(fileSize)/1024/1024)
```
Comment (marten-seemann) [inline integrationtests/rtt_test.go]: I don't think we gain anything from running the test with various file sizes.
I'd rather run the test with different RTT values here, not just 10ms and 75ms.

**Pull request merged**